### PR TITLE
move client documentation from readme

### PR DESCRIPTION
**Pull request by MethodenMann**
### Contributor checklist

- [x] My contribution is **ready to be merged** as is
- [x] I verified the modifications **render properly**
- [x] I used **spell checking**
- [x] I **used the styling and structure aids available** in [Sphinx/ResT](http://www.sphinx-doc.org/en/stable/rest.html). (Links use `` `…`_``, enumerations `#.`, lists `*`, code ``` ``…`` ```/`.. code::`, warnings .. `warning::`, etc.)
- [x] I reread the text keeping in mind that the text has to be **comprehended** fully **by the target audience**

### CODE REVIEW

File: framework/client/contribute.rst [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+**<scope> values**
+
+* If the changes affect a single package the scope is set to package name (e.g. login).
+* If the change is a global or difficult to assign to a single package the parentheses are omitted.
+* If changes affect the mono-repo itself, the scope is set to 'tocco-client'.
+
+Message body
+^^^^^^^^^^^^
+
+* Should consist of a list started with
```
Comment (khanlifts) [inline framework/client/contribute.rst]: Is this the whole sentence?

Comment (khanlifts) [reply framework/client/contribute.rst]: periods at end

File: framework/client/contribute.rst [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+Bug fixing
+^^^^^^^^^^^
+So if a bug is found, let's say in Nice version 2.17, we have to fix this bug in 2.17, 2.18, ... and master.
+Assumed it's a critical bug, otherwise it will just be fixed in master with a pull request branch.
+
+#. Find out the oldest yet supported version of Nice that contains the package with the bug.
+#. Create a fix branch based on the release branch (e.g. ``git checkout -b pr/217/login/bug nice-releases/217``)
+#. Commit fix to branch. Preferably with a regression test to verify the fix.
+#. Creating pull request, wait until approved and rebase into release branch.
```
Comment (khanlifts) [inline framework/client/contribute.rst]: Create a

File: framework/client/contribute.rst [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+#. Checkout release branch and publish the affected package. It's important to not increment the PATCH version for hotfixes in older versions. Chances are that this version already exists on a newer branch. Therefore a --hotfix has to added to the current version. For more info see `Naming`_ chapter.
```
Comment (khanlifts) [inline framework/client/contribute.rst]: to **be** added

File: framework/client/contribute.rst [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+#. Publish an npm tag for the version (e.g. ``npm dist-tag add tocco-login@1.0.2-hotfix217.1 nice217``. With this tag it's possible to change the Nice package.json to this tag instead of a specific version. New fixes will then automatically be installed.
+#. Delete fix branch.
+#. Merging release branch in next version and publish there as well. (this will hopefully be automated sometime)
```
Comment (khanlifts) [inline framework/client/contribute.rst]: Merge

File: framework/client/contribute.rst [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+#. Repeat until hotfix is no more relevant or the bug is fixed in newest version (master).
```
Comment (khanlifts) [inline framework/client/contribute.rst]: in **the** newest version

File: framework/client/development.rst [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+
+.. code-block:: console
+
+    yarn compile:dev --package={PACKAGE_NAME}
+    yarn compile:prod --package={PACKAGE_NAME}
+
+Parameters
+
+=========================== ============================
+``--bundle-analyzer``        Opens BundleAnalyzerPlugin to investigate the bundle sizes
```
Comment (khanlifts) [inline framework/client/development.rst]: period at end

File: framework/client/project-structure.rst [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+``tocco-...`` naming is used in ``package.json``; in folder structure ``tocco-`` prefix can be omitted.
+
+
+Create New Package
+^^^^^^^^^^^^^^^^^^
+A plop template can be used that creates the essentials of a new package.
+
+.. code-block:: console
+
+ yarn plop Package {PACKAGE_NAME}
```
Comment (khanlifts) [inline framework/client/project-structure.rst]: **p**ackage

Comment (MethodenMann) [reply framework/client/project-structure.rst]: plop script is called "Package". So uppercase is correct.

**Pull request merged**